web/app/scripts/custom-reports/custom-report-partial.html

```diff
@@ -7,14 +7,20 @@
                 <span>{{ ::ctl.params.occurred_max | localDateTime : ctl.dateFormat }}</span>
             </h2>
             <h3>
-                Incidents
-                <span ng-if="ctl.params.aggregation_boundary"> for {{ ::ctl.report.table_labels[table.tablekey] }}</span>
-                by {{ ::ctl.rowCategoryLabel }} and {{ ::ctl.colCategoryLabel }}
+                {{ ctl.recordType.plural_label }}
+                <span ng-if="ctl.params.aggregation_boundary">
+                  {{ 'COMMON.FOR' | translate }}
+                  {{ ::ctl.report.table_labels[table.tablekey] }}
+                </span>
+                {{ 'COMMON.BY' | translate }}
```

Comment:
Yeah, I definitely agree, already discussed that (and other such terms) with Tyler. We'll have to discuss with a native speaker and figure out the best way to proceed.